archive/ubiquitous-journey/values-tooling.yaml

```diff
@@ -0,0 +1,224 @@
+destination: &ci_cd_ns labs-ci-cd
```

Comment:
not that I tried but I think we can remove most of the `ignore_differences` from here and `values-extratooling.yaml` file as we put the generic ones inside Argo App template ✨

Reply:
ah sorry! just realize this is under archive:)